File: app/src/main/java/libs/com/rogerio/utils/AccelerometerListener.java
package libs.com.rogerio.utils;
/*
 * Autor: Rogerio C. Santos - rogerio@rogeriocs.com.br
 * http://www.rogeriocs.com.br
 */

public interface AccelerometerListener {

	public void onAccelerationChanged(float x, float y, float z);
	 
	public void onShake(float force);

	
}

File: app/src/main/java/libs/com/rogerio/utils/AccelerometerListener.kt
package libs.com.rogerio.utils

/*
 * Autor: Rogerio C. Santos - rogerio@rogeriocs.com.br
 * http://www.rogeriocs.com.br
 */

interface AccelerometerListener {

    fun onAccelerationChanged(x: Float, y: Float, z: Float)

    fun onShake(force: Float)


}
